gradle-witchcraft-logging/build.gradle

```diff
@@ -45,3 +45,7 @@ pluginBundle {
     description = 'Palantir Witchcraft logging plugins used to improve developer experience building, running, and debugging applications using the witchcraft logging api: https://github.com/palantir/witchcraft-api.'
     tags = ['witchcraft', 'logging', 'idea']
 }
+
+test {
+    systemProperty 'ignoreDeprecations', 'true'
```

Comment:
Complains about using `JavaPluginConvention` - but as far as I can see we don't use `.getConvention` or other `JavaPluginConvention` related stuff anyway. The only thing I think it could be is `AbstractTestTask` extends `ConventionTask`, which doesn't seem a big deal.